tests/functional/healthchecks/test/checkRoutes.js

```diff
@@ -2,6 +2,7 @@ const assert = require('assert');
 const http = require('http');
 const https = require('https');
 const conf = require('../../config.json');
```

Comment:
`Config.js` need to be used instead of  using `config.json` directly. This would enable the tests to pickup any config made available through  the env `S3_CONFIG_FILE`